### DISCUSSION TITLE
Automatic Channel Management

### DISCUSSION THREAD

**Discussion by scarolan** [raspiblitz/raspiblitz#4028] [General]
I dabbled with LND a bit last year but wasn't able to dedicate the time required to keep channels balanced and optimized properly.

Has any progress been made on a 'set and forget' type config? I'd be willing to invest in the network, not expecting to make a huge profit but hopeful the fees could at least cover operational costs for my node.

My main requirement is that it has to be easy - I want to set it up and let it self-manage channels for optimal performance.

**Comment by redmac68** [comment]
You should try LNDg. Breaking even and/or making a small profit running a routing node is unfortunately not easy or 'set and forget' in my experience. LNDg is impressive and definitely helps. Good luck

**Reply by scarolan** [reply]
Cool, I will check it out.
